### ISSUE TITLE
Assa Rogue MFD Singletarget new Build / Prio

### ISSUE THREAD

**Issue by Maxkowa** [Hekili/hekili#1264]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

With Marked for Death skilled the Addon recommands using MFD pre pull and then he uses MFD after Vanish burst window, but it should recommend MFD inside the Vanish Window with Master Assassin Buff.
Means it should looks like this:
Mutilate > Slice and Dice > Mutilate > Garrote > Flagallation > Rapture > Vendetta > Shiv > Vanish > Mutilate > Envenom > Marked for Death > Envenom
basically the goal with this Build is to get a second Envenom into the Master Assasin buff window

### How to Reproduce

1. Enter Game as Assa Rogue
2. Set Hekili options to Enable >
 Display Mode: Single > 
 Toggles: Cooldowns and Covenants: On > 
 Assasination: Marked for Death Combo Points: 3, Allow Vanish when Solo
 3. Start playing the Singletarget opener as recommended and see that he's not recommending MFD inside the Master Assassin buff window

### Snapshot (Link)

https://pastebin.com/94ijsxrg

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/hudPyqjTUQuN8bo5jGr731

### Contact Information

Maxkowa#1901

**Comment by Hekili** [comment]
All priorities will be up to date for 9.1.5.
